lib/server/server.js

```diff
@@ -546,29 +545,34 @@ function execute(port) {
     );
   });
 
-  app.listen(port);
+  // Start LiveReload server.
+  process.env.NODE_ENV = 'development';
+  const server = tinylr();
+  server.listen(constants.LIVE_RELOAD_PORT, function() {
+    console.log(
+      'LiveReload server started on port %d',
+      constants.LIVE_RELOAD_PORT
+    );
+  });
 
-  if (process.env.NODE_ENV !== 'production') {
-    const server = tinylr();
-    server.listen(LIVE_RELOAD_PORT, function() {
-      console.log('Livereload listening on %s...', LIVE_RELOAD_PORT);
-    });
+  // gaze watches some specified dirs and triggers a callback when they change.
+  gaze(
+    [
+      '../docs/**/*', // docs
+      './**/*', // website
```

Comment:
Couldn't quite grok why the dot syntax doesn't work in https://github.com/shama/gaze and minimatch, especially https://github.com/isaacs/minimatch/issues/30. (LiveReload doesn't "fire up" in the terminal like the gif with this syntax here.)

Switched it to just star syntax in the latest commit. Let me know what you think...

Reply:
Ok let's switch it to `**/*`